File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        vowels = {'a':0,'e':0,'i':0,'o':0,'u':0}
        letters = list('abcdefghijiklmnopqrstuvwxyz')
        c = [a for a in letters if a not in vowels.keys()]
        consonants = {key:0 for key in c}
        for k in s:
            if k in vowels.keys():
                vowels[k]+=1
            else:
                consonants[k]+=1

        return max(vowels.values())+max(consonants.values())
        

File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.js
function maxFreqSum(s) {
  const vowels = { a: 0, e: 0, i: 0, o: 0, u: 0 };
  const letters = 'abcdefghijklmnopqrstuvwxyz'.split('');

  // Get consonants (letters not in vowels)
  const consonants = {};
  for (const letter of letters) {
    if (!(letter in vowels)) {
      consonants[letter] = 0;
    }
  }

  // Count frequencies
  for (const k of s) {
    if (k in vowels) {
      vowels[k]++;
    } else if (k in consonants) {
      consonants[k]++;
    }
  }

  // Get max values
  const maxVowel = Math.max(...Object.values(vowels));
  const maxConsonant = Math.max(...Object.values(consonants));

  return maxVowel + maxConsonant;
}
